flows/src/main/java/com/softwaremill/jox/ThrowingConsumer.java

```diff
@@ -0,0 +1,5 @@
+package com.softwaremill.jox;
+
+public interface ThrowingConsumer<T> {
+    void accept(T t) throws Throwable;
```

Comment:
wouldn't `... throws Exception` be enough? (here and in other places)

Reply:
Yes, it will be enough. Changed